packages/conform-react/future/hooks.tsx

```
@@ -1152,98 +1134,102 @@ export function useControl(options?: {
+	return {
+		get defaultPayload() {
+			return parsePayload('defaultPayload', defaultPayloadState);
+		},
+		get payload() {
+			return parsePayload('payload', payloadSnapshot);
+		},
+		get value() {
+			if (typeof payloadSnapshot === 'string') {
+				return payloadSnapshot;
+			}
+		},
```

Biome (2.4.6)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)